Dockerfile

```diff
@@ -18,9 +18,13 @@ RUN apt-get update && \
     lsb-release && \
     apt-get clean all
 
+COPY setup.py .
```

Comment:
we can just copy requirements.txt and install dependencies with that, I don't think copying setup.py is required

also copying requirements.txt to /teuthology_api would be better, maybe something like:
```dockerfile
COPY .teuthology.yaml /root
WORKDIR /teuthology_api
COPY requirements.txt .
RUN pip3 install -r requirements.txt
COPY . /teuthology_api/
RUN pip3 install -e .
```